src/vs/workbench/contrib/chat/browser/chatContentParts/toolInvocationParts/chatResultListSubPart.ts

```diff
@@ -27,17 +32,44 @@ export class ChatResultListSubPart extends BaseChatToolInvocationSubPart {
 	) {
 		super(toolInvocation);
 
-		const collapsibleListPart = this._register(instantiationService.createInstance(
-			ChatCollapsibleListContentPart,
-			toolDetails.map<IChatCollapsibleListItem>(detail => ({
-				kind: 'reference',
-				reference: detail,
-			})),
-			message,
-			context,
-			listPool,
-		));
-		this._register(collapsibleListPart.onDidChangeHeight(() => this._onDidChangeHeight.fire()));
-		this.domNode = collapsibleListPart.domNode;
+		const container = dom.$('div.chat-result-list-subpart');
+
+		const terminalReferences: URI[] = [];
+		const otherReferences: Array<URI | Location> = [];
+		for (const detail of toolDetails) {
+			if (URI.isUri(detail) && detail.scheme === Schemas.vscodeTerminal) {
+				terminalReferences.push(detail);
+			} else {
+				otherReferences.push(detail);
+			}
+		}
+
+		if (terminalReferences.length) {
+			const attachmentEntries = terminalReferences.map(toTerminalCommandVariableEntry);
+			const attachmentsPart = this._register(instantiationService.createInstance(ChatAttachmentsContentPart, {
```

Comment:
When is there more than one? To me the goal of `ChatResultListSubPart` is to show a collapsible list of tool results but it looks like this isn't a collapsible list? Is there a case where there are terminalReferences and otherReferences?